extensions/tokenizers/src/main/python/djl_converter/huggingface_converter.py

```diff
@@ -119,7 +119,8 @@ def save_rust_model(self, model_info, args: Namespace, temp_dir: str,
         if hasattr(config, "model_type"):
             if config.model_type not in [
                     "bert", "camembert", "distilbert", "xlm-roberta",
-                    "roberta", "nomic_bert", "mistral", "qwen2"
+                    "roberta", "nomic_bert", "mistral", "qwen2", "new",
```

Comment:
Add a TODO here, we need monitor if Transformer will give a better model type in few, or if there is conflict.

Reply:
Added